app/views/reviewers/applications/show.html.slim

```diff
@@ -46,7 +46,7 @@
   hr
 
   h2 Mentors' feedback
-  = render partial: 'mentor_feedback', locals: { application: @application }
+  = render partial: 'mentor_feedback', locals: { application: @application, comments: mentor_comments }
```

Comment:
Personal taste, but I really dislike the `helper_method` pattern _(even more than helper modules themselves)_. In my opinion, it makes the already fuzzy process of making things available in views even fuzzier, all that including enough magic to make it really hard to understand what's happening behind the scenes: when is the method invoked? It is an additional request?

I think the best way of staying sane with Rails not being explicit about these things is an instance variable in the controller and then `assigns.fetch('instance-var')` in the view to be absolutely sure both things speak the same language.

Just wanted to have this mentioned ☝️ no need to change it though 😉

Reply:
@klappradla thank you very much for your feedback here.

I'd originally gone with the way you suggested, as this was also present in other tests, but I'm fine with having it this way too. Since the taste seems to be more divisive as @carpodaster originally suggested, I'll be leaving it in for now, if no one minds. Maybe we should strive for consistency down the road, though.

Reply:
Oh, no worries, feel free to go ahead with the helper 👍 

It's just my own personal preference that I hate them - they may actually make sense in some cases ✌️

Reply:
Your personal preference is much appreciate it! I'd love to know why they seem like a bad idea? 

Frankly, if they're as good as the same as helper modules, I think it would be good to write these all consistently.

Reply:
They aren't equal to helper modules. I think they have a closes scope and are only available in views rendered via the controller they're defined in. Plus they can of course also be used in the controller itself.

The reason I dislike them: they blurry the view and hide where things come from 😉 If one assigns an instance variable in the controller it's a bit more obvious from the start where things in the view come from.

But as I said, no biggie, some people like this pattern and there may be obvious reasons to use it - such as the `current_user` helper.

And for view helper modules: I personally think, if it's so much more effort to make things useful for a view, adding a decorator or a view object is an approach that's easier to maintain on the long term then view helpers.
I vaguely recall that things have changed since Rails 5, but before, view helpers were available globally in all views 🙈